src/main/java/org/mockito/internal/creation/bytebuddy/ByteBuddyMockMaker.java

```diff
@@ -22,44 +25,65 @@
  * The programmatic API could look like {@code mock(Final.class, withSettings().finalClasses())}.
  */
 public class ByteBuddyMockMaker implements ClassCreatingMockMaker {
-    private ClassCreatingMockMaker defaultByteBuddyMockMaker = new SubclassByteBuddyMockMaker();
+    private final SubclassByteBuddyMockMaker subclassByteBuddyMockMaker;
+
+    public ByteBuddyMockMaker() {
+        try {
+            subclassByteBuddyMockMaker = new SubclassByteBuddyMockMaker();
+        } catch (NoClassDefFoundError e) {
+            if (e.getMessage() != null && e.getMessage().startsWith("net/bytebuddy/")) {
+                throw new MockitoInitializationException(
```

Comment:
Can we put this exception in Reporter.java so that we dont have to include the same string message twice in both the mockmaker implementations?